internal/service/ssm/parameter_test.go

```diff
@@ -285,10 +285,31 @@ func TestAccSSMParameter_Overwrite_basic(t *testing.T) {
 		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories,
 		CheckDestroy:             testAccCheckParameterDestroy,
 		Steps: []resource.TestStep{
+			{
+
+				PreConfig: func() {
+					conn := acctest.Provider.Meta().(*conns.AWSClient).SSMConn
+
+					input := &ssm.PutParameterInput{
```

Comment:
"test_parameter" here is tightly coupled to the parameter name set through the [testAccParameterConfig_basicOverwrite](https://github.com/bschaatsbergen/terraform-provider-aws/blob/795c5a862a8e1a72bb468a098c8b74d72293dc42/internal/service/ssm/parameter_test.go#L1000), we could optionally remove the `test_parameter` prefix from the config and just set the `name`.